input/example_metadata.json

```
@@ -1,81 +1,280 @@
+    "soil": {
+      "title": "Soil data",
+      "description": "Stop treating soil like dirt!",
+      "path": "soil/example_soil.json",
+      "type": "json",
+      "properties": "g"
+    },
+    "weather": {
+      "title": "Weather data",
+      "description": "Let's turn on the AC",
+      "path": "weather/example_weather.csv",
+      "type": "csv",
+      "properties": "h"
+    }
+  },
+  "properties": {
+    "a": {
+      "calf_num": {
+        "type": "number",
+        "description": "The average number of pre-weaned calves (head)",
+        "minimum": 0,
+        "default": 8
+      },
+      "heiferI_num": {
+        "type": "number",
+        "description": "HeiferI is the number of youngstock post-weaning and before breeding (head)",
+        "minimum": 0,
+        "default": 44
+      },
+      "heiferII_num": {
+        "type": "number",
+        "description": "HeiferII is the number of heifers eligible for breeding and in early pregnancy (head)",
+        "minimum": 0,
+        "default": 38
+      },
+      "heiferIII_num_springers": {
+        "type": "number",
+        "description": "HeiferIII is the number of close-up heifers (head)",
+        "minimum": 0,
+        "default": 5
+      },
+      "cow_num": {
+        "type": "number",
+        "description": "The number of dry and lactating cows (head)",
+        "minimum": 0,
+        "default": 100
+      },
+      "herd_num": {
+        "type": "number",
+        "description": "The number of lactating and milking cows on farm (head)",
+        "minimum": 0,
+        "default": 187
+      },
+      "herd_init": {
+        "type": "boolean",
+        "description": "Whether or not to initialize a herd if the milk prod and body weight numbers are exceedingly abnormal",
+        "default": false
+      },
+      "breed": {
+        "type": "string",
+        "description": "The predominant breed of the herd (Holstein - HO or Jersey - JE)",
+        "default": "HO",
+        "minimum_length": 2,
+        "maximum_length": 2
       }
     },
-    "properties": {
-      "a": {},
-      "b": {},
-      "c": {},
-      "d": {},
-      "e": {},
-      "f": {},
-      "g": {},
-      "h": {}
+    "b": {
+      "crop_species": {
+        "type": "string",
+        "description": "the name of the crop being grown",
+        "minimum_length": 1,
+        "maximum_length": 50
+      },
+      "planting_years": {
+        "type": "array",
+        "description": "Years in which the planting occurs",
+        "minimum_length": 1
+      },
+      "planting_days": {
+        "type": "array",
+        "description": "Julian day(s) of year to plant",
+        "minimum_length": 1
+      },
+      "harvest_years": {
+        "type": "array",
+        "description": "Years in which the harvesting occurs",
+        "minimum_length": 1
+      },
+      "harvest_days": {
+        "type": "array",
+        "description": "Julian day(s) of year to harvest",
+        "minimum_length": 1
+      }
+
```

A to Z, the same applies to other blobs.
```suggestion
    "b": {
       "crop_species": {
        "type": "string",
        "description": "the name of the crop being grown",
        "minimum_length": 1,
        "maximum_length": 50
      },
      "harvest_days": {
        "type": "array",
        "description": "Julian day(s) of year to harvest",
        "minimum_length": 1
      },
      "harvest_years": {
        "type": "array",
        "description": "Years in which the harvesting occurs",
        "minimum_length": 1
      },
      "planting_days": {
        "type": "array",
        "description": "Julian day(s) of year to plant",
        "minimum_length": 1
      },
      "planting_years": {
        "type": "array",
        "description": "Years in which the planting occurs",
        "minimum_length": 1
      }
```